src/webportal/src/app/job/job-view/fabric/JobList/Ordering.js

```diff
@@ -0,0 +1,64 @@
+import {getModified, getDuration, getStatusIndex} from './utils';
+
+export default class Ordering {
+  /**
+   * @param {"name" | "modified" | "user" | "duration" | “virtualCluster” | "status" | undefined} field
```

Comment:
The curved quotes may be typos

Reply:
It's just for better type intellisence.

https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#jsdoc-types-are-used-for-type-information
https://www.typescriptlang.org/docs/handbook/advanced-types.html#string-literal-types